tools/pvacvector/run.py

```diff
@@ -369,6 +372,46 @@ def identify_problematic_peptides(Paths, seq_dict):
             problematic_start.add(node)
     return (problematic_start, problematic_end)
 
+def create_dna_backtranslation(results_file, dna_results_file):
+    record = SeqIO.read(results_file, 'fasta')
+    seq_num = record.id
+    peptide = str(record.seq)
+    params = {
+        u'email': 'susanna.kiwala@wustl.edu',
```

Comment:
Does it make sense for you to take personal responsibility for everyone using this tool?

Reply:
Ah no, I meant to test it without an email to see if it would work. If email is indeed required we should make a new one for this use case.